client/components/filters/filters.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var app = angular.module('app');
+
+app.filter('parseString', function() {
```

Comment:
`parseString` doesn't really describe the function of this filter. `truncate` would probably be a better name.